ios/Classes/Embeds/AppcuesFrameViewFactory.swift

```diff
@@ -0,0 +1,195 @@
+import Flutter
+import UIKit
+import AppcuesKit
+
+class AppcuesFrameViewFactory: NSObject, FlutterPlatformViewFactory {
+    private let messenger: FlutterBinaryMessenger
+    private let plugin: SwiftAppcuesFlutterPlugin
+
+    init(plugin: SwiftAppcuesFlutterPlugin, messenger: FlutterBinaryMessenger) {
+        self.plugin = plugin
+        self.messenger = messenger
+        super.init()
+    }
+
+    func create(
+        withFrame frame: CGRect,
+        viewIdentifier viewId: Int64,
+        arguments args: Any?
+    ) -> FlutterPlatformView {
+        return AppcuesPlatformView(
+            plugin: plugin,
+            frame: frame,
+            viewIdentifier: viewId,
+            arguments: args,
+            binaryMessenger: messenger)
+    }
+
+    /// Implementing this method is only necessary when the `arguments` in `createWithFrame` is not `nil`.
+    public func createArgsCodec() -> FlutterMessageCodec & NSObjectProtocol {
+          return FlutterStandardMessageCodec.sharedInstance()
+    }
+}
+
+class AppcuesPlatformView: NSObject, FlutterPlatformView {
+    private var _view: UIView
+
+    init(
+        plugin: SwiftAppcuesFlutterPlugin,
+        frame: CGRect,
+        viewIdentifier viewId: Int64,
+        arguments args: Any?,
+        binaryMessenger messenger: FlutterBinaryMessenger?
+    ) {
+        let args = args as? [String: Any]
+
+        _view = WrapperView(
+            plugin: plugin,
+            binaryMessenger: messenger,
+            frameID: args?["frameId"] as? String,
```

Comment:
`frameID` comes through the arguments from the Flutter side and is pulled out here